string scene_name
sensor_msgs/Image image